### DISCUSSION TITLE
通过cdnjs加速插件安装

### DISCUSSION THREAD

**Discussion by BugKun** [the1812/Bilibili-Evolved#3519] [功能建议 / Ideas]
在国内jsdelivr不稳定的情况下，建议可以通过[cdnjs](https://github.com/cdnjs/cdnjs)加速插件安装，国内有很多cdnjs的镜像加速，如字节跳动的

https://cdn.bytedance.com/

**Comment by the1812** [comment]
v2.3.1 后只使用 GitHub Raw 了, 需要使用其他 CDN 请自行 fork
